src/components/geojson-map.vue

```diff
@@ -80,9 +80,6 @@ const { redAlert, config, buildMode } = inject('container');
 // eslint-disable-next-line no-console
 const log = config.devTools ? console.log.bind(console) : noop;
 
-// Constants
-const animationDuration = 1000;
```

Comment:
This PR introduces a new constant, `viewPadding`. I thought about defining `viewPadding` here next to the only other constant, `animationDuration`. However, I think I prefer defining these constants closer to the code that references them. These constants are first used in and are closely tied to the section about the view, so I moved them there.

No strong feelings about this, I'm also happy to define `viewPadding` here.